modules/compat.lua

```diff
@@ -0,0 +1,98 @@
+-- Item conversion borrowed from ox_inventory: https://github.com/overextended/ox_inventory/blob/52d99285eef1dc7df31c084679db3fdf6b1c0150/modules/items/server.lua#L115-L217
+-- Copyright (c) 2023 Overextended
+
+return {
```

Comment:
The modules folder is for publicly available libraries. This should be moved somewhere else. Probably within the bridge.

Reply:
yeah i just threw it in there as a start. i'll move it